hydromt_wflow/workflows/river.py

```diff
@@ -121,7 +128,23 @@ def river(
     # set mean length at pits when taking the downstream length
     if channel_dir == "down":
         rivlen.flat[flwdir.idxs_pit] = rivlen_avg
-
+    # smooth river length if min_rivlen is set
+    if min_rivlen != None:
+        logger.debug(f"Smooth river length (min length: {min_rivlen:.0f} m).")
+        flwdir_model = flw.flwdir_from_da(ds_model["flwdir"], mask=True)
+        rivlen_s = flwdir_model.smooth_rivlen(
+            rivlen=rivlen,
+            min_rivlen=min_rivlen,
+            smooth_cells=smooth_cells,
+            mask=riv_mask,
+        )
+        assert_equal = np.isclose(
```

Comment:
I think this is a good test, but wonder if this shouldn't be moved to the unit tests of pyflwdir instead?

Reply:
I agree, the test has been removed in commit https://github.com/Deltares/hydromt_wflow/pull/92/commits/298d83d2792e89bd975253b2d6f62d13afb50be5.